<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>10</width>
    <height>7</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="LE">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Bu1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:17px;
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="iconSize">
     <size>
      <width>56</width>
      <height>12</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextOnly</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Bu2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bu3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bu4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bu5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bu6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bu7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bu8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bu15">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bu14">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bu13">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bu12">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bu0">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bu10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bu9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bu16">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 17px;</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Esc">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">size:17px;</string>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">size:17px;</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="left_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">size:17px;</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">size: 17px;</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="MyButton" name="Save_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Click to save</string>
    </property>
   </widget>
   <widget class="MyButton" name="Save_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Click to save</string>
    </property>
   </widget>
   <widget class="MyButton" name="Save_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Click to save</string>
    </property>
   </widget>
   <widget class="MyButton" name="Save_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Click to save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
border:  0px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cos_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tan_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arcsin_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:8px;</string>
    </property>
    <property name="text">
     <string>arcsin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arccos_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 8px;</string>
    </property>
    <property name="text">
     <string>arccos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arctan_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 8px;</string>
    </property>
    <property name="text">
     <string>arctan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ln_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pi_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:15px;</string>
    </property>
    <property name="text">
     <string>π</string>
    </property>
   </widget>
   <widget class="QPushButton" name="e_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:15px;</string>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawbutton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>draw</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
border:  0px;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>337</y>
      <width>1910</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Click to change the input area</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
border:  0px;</string>
    </property>
    <property name="text">
     <string> y=</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="x_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>210</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 11px;</string>
    </property>
    <property name="text">
     <string>1/x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mod_Button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:8px</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <zorder>Bu9</zorder>
   <zorder>LE</zorder>
   <zorder>Bu1</zorder>
   <zorder>toolButton</zorder>
   <zorder>Bu2</zorder>
   <zorder>Bu3</zorder>
   <zorder>Bu4</zorder>
   <zorder>Bu5</zorder>
   <zorder>Bu6</zorder>
   <zorder>Bu7</zorder>
   <zorder>Bu8</zorder>
   <zorder>Bu15</zorder>
   <zorder>Bu14</zorder>
   <zorder>Bu13</zorder>
   <zorder>Bu12</zorder>
   <zorder>Bu0</zorder>
   <zorder>Bu10</zorder>
   <zorder>Bu16</zorder>
   <zorder>Esc</zorder>
   <zorder>pushButton</zorder>
   <zorder>right_button</zorder>
   <zorder>left_button</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>Save_1</zorder>
   <zorder>Save_4</zorder>
   <zorder>Save_2</zorder>
   <zorder>Save_3</zorder>
   <zorder>lineEdit</zorder>
   <zorder>sin_button</zorder>
   <zorder>cos_button</zorder>
   <zorder>tan_button</zorder>
   <zorder>arcsin_button</zorder>
   <zorder>arccos_button</zorder>
   <zorder>arctan_button</zorder>
   <zorder>ln_button</zorder>
   <zorder>pi_button</zorder>
   <zorder>e_button</zorder>
   <zorder>drawbutton</zorder>
   <zorder>LE2</zorder>
   <zorder>radioButton</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>x_button</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>mod_Button</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>343</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>你好</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyButton</class>
   <extends>QPushButton</extends>
   <header>mybutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
